lib/all_rooms.rb

```diff
@@ -0,0 +1,38 @@
+module Hotel
+    class AllRooms
```

Comment:
This is a small thing to prepare for Rails, but we generally don't name a class in the plural. Since classes are meant to be instantiated, we name them in the singular, as they represent the 'bones' of one entity.